packages/release-please/src/server.ts

```diff
@@ -12,22 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// eslint-disable-next-line node/no-extraneous-import
-import {Probot} from 'probot';
 import {GCFBootstrapper} from 'gcf-utils';
+import {api} from './release-please';
 
 const bootstrap = new GCFBootstrapper({
-  taskTargetEnvironment: 'functions',
+  taskTargetEnvironment: 'run',
 });
-
-// We only need to deploy gcf-utils in the frontend server because
-// only thing it does is to enqueue a task for the release-please
-// backend. Thus we deploy a dummy empty bot code.
-
```

Comment:
So are you planning to use this file for both frontend and backend?

Reply:
Yes, we are currently running a frontend, but we will have a single service that handles both.

Reply:
Now we are running separate frontend/backend.